babel.config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
- [ ] It would be better if our `babel.config.js` file is configured properly in order for the successful running of the app. Kindly check this [link](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a) so we can properly configure the app as expected. Also take note of the file extension name